docs/components/concepts/access-control/authorizations.md

```diff
@@ -0,0 +1,150 @@
+| **Decision Definition**              | `*`, `decisionId: order_decision`                       | `CREATE_DECISION_INSTANCE`, `READ_DECISION_DEFINITION`, `READ_DECISION_INSTANCE`, `DELETE_DECISION_INSTANCE`               |
+| **Decision Requirements Definition** | `*`, `decisionRequirementsDefinitionId: order_decision` | `CREATE`, `READ`, `UPDATE`                                                                                                 |
+| **Group**                            | `*`, `groupId: accounting`                              | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Mapping Rules**                    | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Message**                          | `*`                                                     | `CREATE`, `READ`                                                                                                           |
+| **Process Definition**               | `*`, `bpmnProcessId: order_process`                     | `CREATE_PROCESS_INSTANCE`, `READ_PROCESS_DEFINITION`, `READ_PROCESS_INSTANCE`, `READ_USER_TASK`, `UPDATE_PROCESS_INSTANCE` |
+| **Resource**                         | `*`                                                     | `CREATE`, `DELETE_DRD`, `DELETE_FORM`, `DELETE_PROCESS`                                                                    |
+| **Roles**                            | `*`                                                     | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **System**                           | `*`                                                     | `READ`, `UPDATE`                                                                                                           |
+| **Tenants**                          | `*`, `tenantId:tenantA`                                 | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+| **Users**                            | `*`, `username: felix.mueller`                          | `CREATE`, `READ`, `UPDATE`, `DELETE`                                                                                       |
+
+## Default Roles
+
+Camunda provides predefined roles to simplify access management:
+
+| Role Name       | Purpose                                                                         | Typical Authorizations                                                                                                                              |
```

Comment:
❓ Do we want the role name or the role id here?

If it's role name it should be:
- `Readonly Admin`
- `Admin`
- `RPA`
- `Connectors`

If it's role id it should be:
- `readonly-admin`
- `admin`
- `rpa`
- `connectos`